src/Raven.Studio/wwwroot/Content/css/basic-layout.less

```diff
@@ -1,4 +1,5 @@
 ﻿@menu-width: 280px;
+@added-width: 120px;
```

Comment:
why added width ? 
Maybe call it right panel width?

Reply:
why 120px ? Isn't it too much? Especially when document doesn't have attachments, counters and revisions? AFAIK we talked about adding 70px?

Reply:
todo:  introduce right-panel-width 
for right panel & notification center

Reply:
done